components/centraldashboard/webpack.config.js

```diff
@@ -81,19 +81,19 @@ module.exports = {
                     }
                 }
             },
-            {
-                enforce: 'pre',
-                test: /\.js$/,
-                exclude: NODE_MODULES,
-                use: {
-                    loader: 'eslint-loader',
-                    options: {
-                        extends: ['eslint:recommended', 'google'],
-                        failOnError: true,
-                        fix: true
-                    }
-                }
-            },
+            // {
```

Comment:
I'm not sure why this would fail. It builds properly in the Docker container so that seems to indicate that it might be something unique to your machine. What kind of errors are you seeing?

Reply:
> Missing eslint configuration

Reply:
Hmm and that's still the case after deleting `node_modules/` and doing a clean `npm install`? The required configs should be in devDependencies.

```
    "eslint": "^5.14.1",
    "eslint-config-google": "^0.12.0",
    "eslint-loader": "^2.1.2",
```

Reply:
This is the error was:
```
ERROR in (webpack)-dev-server/client?http://localhost:8081
[Webpack] Module build failed (from <redacted>/Google/Kubeflow/kubeflow/components/centraldashboard/node_modules/eslint-loader/index.js):
[Webpack] Error: No ESLint configuration found.
[Webpack]     at Config.getLocalConfigHierarchy (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint\lib\config.js:268:39)
[Webpack]     at Config.getConfigHierarchy (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint\lib\config.js:192:43)
[Webpack]     at Config.getConfigVector (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint\lib\config.js:299:21)
[Webpack]     at Config.getConfig (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint\lib\config.js:342:29)
[Webpack]     at processText (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint\lib\cli-engine.js:181:33)
[Webpack]     at CLIEngine.executeOnText (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint\lib\cli-engine.js:690:40)
[Webpack]     at lint (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint-loader\index.js:263:17)
[Webpack]     at Object.module.exports (<redacted>\Google\Kubeflow\kubeflow\components\centraldashboard\node_modules\eslint-loader\index.js:258:21)
[Webpack]  @ multi (webpack)-dev-server/client?http://localhost:8081 <redacted>/Google/Kubeflow/kubeflow/components/centraldashboard/public/index.js main[0]
```

Reply:
As discussed, this seems like a Windows-platform specific issue since the build succeeds on Linux and within Docker during its build process. Please leave the eslint-loader section in place.